packages/openebs/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,9 +1,12 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
-@@ -17,3 +17,6 @@
- sources:
- - https://github.com/openebs/openebs
- version: 1.12.3
+@@ -1,5 +1,9 @@
+ apiVersion: v2
+ appVersion: 2.11.0
++kubeVersion: '>=1.13.0-0'
 +annotations:
 +  catalog.cattle.io/certified: partner
 +  catalog.cattle.io/release-name: openebs
```

Comment:
Please add the display name annotation for the name you want to be displayed in the UI. For example: `catalog.cattle.io/display-name: OpenEBS`. Do not modify this file directly, follow the steps to add it to the patch using the make commands.